Python/pywarpx/_libwarpx.py

```diff
@@ -334,7 +336,7 @@ def finalize(finalize_mpi=1):
     the end of your script.
 
     '''
-    if warpx_initialized == True:
+    if warpx_initialized:
```

Comment:
Would this be needed here, too?
```suggestion
    global warpx_initialized
    if warpx_initialized:
```

Reply:
Yeah, the Stack Overflow link above says that since functions will access module variables after checking local variables for read operations, python doesn't require the global call here. The problem is the default for write is to create the local variable, not modify the module var.

I'm indifferent on a style choice to specify it or not.

Reply:
I think since we generally try to avoid globals and this is kind of an exception for a module global, I would actually add it here to avoid confusion.